scripts/update-preview.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# See https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail
+set -euo pipefail
+env
+
+VERSION=$1
+
+echo "Update preview app version"
+cd preview
+
+echo "Amend Gemfile to use the right version"
+sed -i.bak "s/gem 'ably-ui', '.*', require/gem 'ably-ui', '${VERSION}', require/" Gemfile
```

Comment:
hey @kennethkalmer this is to make the command behave consistently with the built-in OSX sed and the one on CircleCI (ubuntu)/

With the exact above command, if I change it to `-i`, it will work on CircleCi but fail on OSX.

Reply:
TIL